src/components/sidebar/sidebar.style.js

```diff
@@ -42,8 +44,20 @@ const SidebarStyle = styled.div`
   }
 `;
 
+export const SidebarContentBox = styled(Box)``;
```

Comment:
This whole styled component is unnecessary, we could just have 
```jsx
<Box 
  mr={-4}
  pr={4}
  height="calc(100% - 64px)"
  scrollVariant= "light"
  overflow="auto"
>
  {children} 
</Box>
```

in `sidebar.component.js`. The user can't override any of these so they don't need to be default props, just hardcoded ones